core/chain-configs/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+mod client_config;
+mod genesis_config;
+
+pub use client_config::ClientConfig;
+pub use genesis_config::GenesisConfig;
+pub use genesis_config::CONFIG_VERSION as GENESIS_CONFIG_VERSION;
```

Comment:
Why `GENESIS_CONFIG_VERSION` and `PROTOCOL_VERSION` are different values?

Reply:
Config is for migrations